testing/integration_test.go

```diff
@@ -199,6 +200,11 @@ func TestAgentIntegration(t *testing.T) {
 		ComputeService: computeService,
 	}
 
+	wantProfiles := []profileSummary{
```

Comment:
The current checks appear to only verify that there is any sample for busyLoop and any sample for busybench.ts. It would be good to have a more strict check that there are samples in busyLoop function and busybench.ts source file.

Reply:
This is WIP.
Adding a function to support this to proftest.go (https://code-review.googlesource.com/c/gocloud/+/36950)

Reply:
I also don't mind this enhancement to come later.

Reply:
@nolanmar511 This is done?

Reply:
Yes! This is done. 

I lost this comment, sorry.